openid-4-verifiable-presentations-1_0.md

```diff
@@ -232,7 +232,7 @@ Presentation of Verifiable Credentials using OpenID for Verifiable Presentations
 
 The Authorization Request follows the definition given in [@!RFC6749] taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
-The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
+The Verifier MAY send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101]. Additionally, the request can be an object containing only a subset of parameters needed to, in a subsequent step, request the creation of a request object from the Verifier through an HTTPS POST request via a newly introduced `create request endpoint`. 
```

Comment:
Does this mean the first request object can be a partial JAR and the subsequent request object contains the remaining authorization request parameters?

Or does it mean it can be either a JAR, or something else that is something like a partial authorization request object and  the remaining parameters must be obtained from the create_request_uri endpoint? 

What das the "Additionally" refer to?

Reply:
I think this concern gets resolved by my suggestions below. So, I"m resolving this for now.